HW03/aleksei_zhuchkov/aleksei_zhuchkov_hw_03_t_04.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task4(data_log = 'error')
+    raise ArgumentError if data_log == 'error'
+
+    count_digits = data_log.scan(/\d/).flatten.count
```

Comment:
you can use only ```count``` and regular value

Reply:
Tnkx,  I corrected my code https://github.com/labs-ruby/homeworks-2022/pull/63/commits/e69fe35bcb32de79d36501e78ad4e534d74c422c

Reply:
I did it